pkg/apis/extensions/register.go

```diff
@@ -42,8 +42,10 @@ func Resource(resource string) schema.GroupResource {
 }
 
 var (
+	// SchemeBuilder is the new scheme builder
```

Comment:
Nit: Could you maybe include context on why we have a single `AddToScheme` and `SchemeBuilder` that we share across the entire package?